### PR TITLE
Added note about having proper sign-off on commit

### PR DESCRIPTION
**Pull request by lehors**
Lab proposal submitters consistently stumble over the lack of sign-off. This note will hopefully
provide them with proper guidance.

Signed-off-by: Arnaud J Le Hors <lehors@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,24 @@ for additional information.
...
+1. Fork the [hyperledger-labs.github.io](https://github.com/hyperledger-labs/hyperledger-labs.github.io) repository.
 
-Fill out the [Proposal Template](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/proposal-template.md)
+2. Fill out the [Proposal Template](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/proposal-template.md)
 and save it into the labs subdirectory under the name of your lab,
 such as mynewlab.md. It is expected that your lab repository will have
 the same name so keep that in mind.
 
-Submit a Pull Request.
+Make sure your commit has proper sign-off. This means that your commit
```
Comment (tkuhrt) [inline README.md]: should this be a step 3. Commit the newly created proposal (even though it is obvious to us)?

**Comment by trbs** [comment]
Maybe also put in a link to Tracy's https://gist.github.com/tkuhrt/10211ae0a26a91a8c030d00344f7d11b ?

**Comment by trbs** [comment]
I would also like to put in a few words on the title and description of the PR itself. Since maybe people submit them with empty description.

Or better yet, shall I create a PR with a `.github/pull_request_template.md` file ?

File: README.md (hunk 1 above)
Comment (lehors) [reply README.md]: Well, I see your point but is it really possible someone will want to create a lab and not know that they need to make a commit to make a PR?
I'd say that if that were the case they probably wouldn't know how to make a PR either. Then, where do we stop?

**Comment by lehors** [comment]
> I would also like to put in a few words on the title and description of the PR itself. Since maybe people submit them with empty description.
> 
> Or better yet, shall I create a PR with a `.github/pull_request_template.md` file ?



> I would also like to put in a few words on the title and description of the PR itself. Since maybe people submit them with empty description.
> 
> Or better yet, shall I create a PR with a `.github/pull_request_template.md` file ?

That's a good idea!

File: README.md (hunk 1 above)
Comment (lehors) [reply README.md]: Nevermind, I'll make the note a step 3. :-)

**Comment by tkuhrt** [comment]
FYI, I created [this new gist](https://gist.github.com/tkuhrt/2cb6ae09fa6c122674766d2bb9239cdb) specifically for hyperledger-labs. Please review and see if I missed anything with my copy/paste/correct.

**Comment by vipinsun** [comment]
lgtm

**Comment by vipinsun** [comment]
lgtm

**Pull request merged**